src/rprblender/engine/context.py

```diff
@@ -452,12 +455,14 @@ def set_material_node_as_material(self, key, material_node):
 
     def create_image_file(self, key, filepath):
         image = pyrpr.ImageFile(self.context, filepath)
+        image.set_compression = self.compress_textures
```

Comment:
set_compression is a method in Image class. Seems must be image.set_compression(self.compress_textures)